operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -959,6 +984,68 @@ private long delayUntilNextRetry(@Nonnull DomainPresenceInfo domainPresenceInfo)
 
   }
 
+  private class ClusterPlan extends Plan {
+
+    public ClusterPlan(MakeRightClusterOperation operation, DomainProcessorDelegate delegate) {
+      super(operation, delegate);
+    }
+
+    @Override
+    public CompletionCallback createCompletionCallback() {
+      return new ClusterPlanCompletionCallback();
+    }
+
+    class ClusterPlanCompletionCallback implements CompletionCallback {
+
+      @Override
+      public void onCompletion(Packet packet) {
+        // no op
```

Comment:
Does this need to handle kubernetes failures?

Reply:
The Kubernetes failures on creating an event is handled in EventHelper already.

Reply:
ok.